client/js/lounge.js

```diff
@@ -829,6 +829,20 @@ $(function() {
 			return;
 		}
 
+		if (text.substring(0, 5) === "/join") {
```

Comment:
Updated. A cleaner and a more efficient solution might require refactoring the event handler function.